frontend/src/components/RecentReleases.tsx

```diff
@@ -67,11 +67,17 @@ const RecentReleases: React.FC<RecentReleasesProps> = ({
                   <h3 className="flex-1 overflow-hidden text-ellipsis whitespace-nowrap font-semibold">
                     <Link
                       className="text-blue-400 hover:underline"
-                      href={item?.url || '/'}
+                      href={
+                        item?.url && item?.url !== '/'
+                          ? item.url
+                          : item.organizationName && item.repositoryName && item.tagName
+                            ? `/organizations/${encodeURIComponent(item.organizationName)}/repositories/${encodeURIComponent(item.repositoryName)}/releases/tag/${encodeURIComponent(item.tagName)}`
```

Comment:
We don't have any pages at that address 🤔 :
![Screenshot 2025-06-07 at 9 20 04 PM](https://github.com/user-attachments/assets/386a342e-0414-451a-b421-3335534f55e0)

I believe this should lead to the release page on GitHub